includes/tracks/class-wc-tracks.php

```diff
@@ -65,6 +77,16 @@ public static function get_identity( $user_id ) {
 
 	}
 
+	public static function update_revenue_cache() {
+		update_option( self::REVENUE_CACHE_OPTION, WC_Tracker::get_order_totals() );
+	}
+
+	public static function get_total_revenue() {
```

Comment:
Also we could rename this method to `get_order_totals`, so in the future we can use for more items.

Reply:
It's only returning the `gross` value for now though.. are you wanting to see the return value change as well or can we punt that for when its needed?

Reply:
Let's make generic enough to be used for more, since the `WC_Tracker::get_order_totals();` already returns an array in case need to return more data.